doc/api/test.md

```diff
@@ -1521,6 +1521,14 @@ Emitted when all subtests have completed for a given test.
 
 Emitted when a test starts.
 
+### Event: `'test:stdout'`
```

Comment:
Won't we need `test:stderr` as well?

Reply:
I think it's better to have a separate event for `stderr` because `stdout` and `stderr` is fundamentally different in usage although both are `outputstream`. What do you think?